documentation/testing/test-strategy.md

```diff
@@ -1,219 +1,174 @@
-| -          | Local          | -             | Unit                |
-| TBC        | Dev            | Development   | Manual acceptance   |
-| TBC        | Dev2           | Development   | Feature/e2e         |
-| TBC        | Test           | Test          | Non-functional tests |
-| TBC        | Production/Live | Production    | -                   |
-| TBC        | Showcase       | Production    | Security(flexible)  |
-
-## Local
-The unit and integration testing is performed by the developers to test their code.
+On an ad-hoc or release basis, exploratory testing is carried out to ensure that the service integrates together correctly and works as expected on all supported devices.
 
-## Development
-** environments details will be added here ** 
+This testing is carried out by test engineer(s) and/or developers when it is deemed that there is a high risk of integration or regression issues. This shall also be used for checking the compatibility of the site with all required browsers and operating systems. Test charters are created outlining the areas of the service to be tested.
 
+### Testing Requirements
 
-## Path to live
+#### Supported Browsers
+We will be focusing on Chrome browser on Windows and MacOs and Edge on windows considering the insights we have from the previous system. The rest of the browsers shown below will be checked on adhoc basis. 
 
-//TODO: Create a path to live diagram, that visualises the stage gates from ideation through to release
+| Operating System | Browser                      | Versions                     |
+|------------------|------------------------------|------------------------------|
+| Windows          | Edge                         | Latest                       |
+|                  | Google Chrome                | Latest                       |
+|                  | Mozilla Firefox              | Latest                       |
+| macOS            | Safari                       | 12 and later                 |
+|                  | Google Chrome                | Latest                       |
+|                  | Mozilla Firefox              | Latest                       |
+| iOS              | Safari for iOS               | 12.1 and later               |
+|                  | Google Chrome                | Latest                       |
+| Android          | Google Chrome                | Latest                       |
+|                  | Samsung Internet             | Latest                       |
 
-![Path to Live](testing/images/path-to-live.jpg)
+Any issues or discrepancies between browsers will be raised as defects and then assessed for priority and severity.
+#### Environments
 
+| Env Prefix | Env Name       | Testing             |
+|------------|----------------|---------------------|
+| -          | Local          | -                   |
+| d01        | Dev            | Manual acceptance   |
+| d02        | Dev2           | Feature/e2e         |
```

Comment:
All automated testing will take place in d02 (environment has been named "automated-test")

Reply:
yes forgot to update this section. updated now.